Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM_2nd.cpp

```diff
@@ -153,6 +158,31 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian_2nd(
                         Hz_eff += MacroscopicProperties::face_avg_to_face(i, j, k, 0, amrex::IntVect(0, 0, 1), amrex::IntVect(1, 0, 0), Hz_old);
                     }
 
+                    int mag_exchange = 0;
+                    if (mag_exchange == 1){
```

Comment:
We will ultimately want to test to make sure the exchange_constant !=0 and M_s != 0